src/DatabaseSettingStore.php

```diff
@@ -222,7 +226,11 @@ protected function prepareInsertData(array $data)
 	 */
 	protected function read()
 	{
-		return $this->parseReadData($this->newQuery()->get());
+		$result = Cache::remember('settings', Config::get('cache.ttl', 60), function () {
```

Comment:
There is nothing handling `Cache` facade as we unit test this without Laravel. You can always decorate this class with a `CachingSettingStore`.